Develop an SQL schema for SOPs

```diff
@@ -0,0 +1,4351 @@
+-- Dependencies: 249
+-- Name: fittedFragments_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: postgres
+--
+
+ALTER SEQUENCE public."fittedFragments_id_seq" OWNED BY public."fittedFragments".id;
+
+
+--
+-- TOC entry 230 (class 1259 OID 17868)
+-- Name: fittedMeasuredFluxResiduals; Type: TABLE; Schema: public; Owner: postgres
+--
+
+CREATE TABLE public."fittedMeasuredFluxResiduals" (
+    "id " integer NOT NULL,
+    simulation_id text,
+    "simulation_dateAndTime" timestamp(6) without time zone,
+    experiment_id text,
+    " sample_name_abbreviation" text,
+    time_point text,
+    rxn_id text,
+    res_data double precision,
+    res_esens double precision,
+    res_fit double precision,
+    res_msens double precision,
+    res_peak text,
+    res_stdev double precision,
+    res_val double precision,
+    used boolean,
+    comment text
+);
+
+
+ALTER TABLE public."fittedMeasuredFluxResiduals" OWNER TO postgres;
+
+--
+-- TOC entry 251 (class 1259 OID 18018)
+-- Name: fittedMeasuredFluxResiduals_id _seq; Type: SEQUENCE; Schema: public; Owner: postgres
+--
+
+CREATE SEQUENCE public."fittedMeasuredFluxResiduals_id _seq"
+    AS integer
+    START WITH 1
+    INCREMENT BY 1
+    NO MINVALUE
+    NO MAXVALUE
+    CACHE 1;
+
+
+ALTER TABLE public."fittedMeasuredFluxResiduals_id _seq" OWNER TO postgres;
+
+--
+-- TOC entry 3148 (class 0 OID 0)
+-- Dependencies: 251
+-- Name: fittedMeasuredFluxResiduals_id _seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: postgres
+--
+
+ALTER SEQUENCE public."fittedMeasuredFluxResiduals_id _seq" OWNED BY public."fittedMeasuredFluxResiduals"."id ";
+
+
+--
+-- TOC entry 231 (class 1259 OID 17871)
+-- Name: fittedMeasuredFluxes; Type: TABLE; Schema: public; Owner: postgres
+--
+
+CREATE TABLE public."fittedMeasuredFluxes" (
+    id integer NOT NULL,
+    simulation_id text,
+    "simulation_dateAndTime" timestamp(6) without time zone,
+    experiment_id text,
+    sample_name_abbreviation text,
+    " rxn_id" text,
+    fitted_sres double precision,
+    used boolean,
+    comment text
+);
+
+
+ALTER TABLE public."fittedMeasuredFluxes" OWNER TO postgres;
+
+--
+-- TOC entry 252 (class 1259 OID 18030)
+-- Name: fittedMeasuredFluxes_id_seq; Type: SEQUENCE; Schema: public; Owner: postgres
+--
+
+CREATE SEQUENCE public."fittedMeasuredFluxes_id_seq"
+    AS integer
+    START WITH 1
+    INCREMENT BY 1
+    NO MINVALUE
+    NO MAXVALUE
+    CACHE 1;
+
+
+ALTER TABLE public."fittedMeasuredFluxes_id_seq" OWNER TO postgres;
+
+--
+-- TOC entry 3149 (class 0 OID 0)
+-- Dependencies: 252
+-- Name: fittedMeasuredFluxes_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: postgres
+--
+
+ALTER SEQUENCE public."fittedMeasuredFluxes_id_seq" OWNED BY public."fittedMeasuredFluxes".id;
+
+
+--
+-- TOC entry 232 (class 1259 OID 17874)
+-- Name: fittedMeasuredFragmentResiduals; Type: TABLE; Schema: public; Owner: postgres
+--
+
+CREATE TABLE public."fittedMeasuredFragmentResiduals" (
+    " id" integer NOT NULL,
```

Comment:
Please remove this extra whitespace and those below

Reply:
done